session.go

```diff
@@ -64,6 +64,9 @@ type Session interface {
 	// of whether or not a PTY was accepted for this session.
 	Pty() (Pty, <-chan Window, bool)
 
+	// Sftp returns a boolean, when subsystem is sftp, return true
+	Sftp() bool
```

Comment:
Maybe this should be `Subsystem(string) bool` so it can be used for other subsystems.

Reply:
I have already modified,please review it, thank you.